### PR TITLE
VACMS-19557: DUW v2 Results updates

### PR DESCRIPTION
**Pull request by chriskim2311**
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to TeamSites and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary
Updates content on the results screen for form 149

## Related issue(s)
[#19557](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/19557)

## Testing done
Tested locally against staging to check that the proper item number shows.
How_To_Apply_For_A_Discharge_Upgrade___Veterans_Affairs

## Screenshots
How_To_Apply_For_A_Discharge_Upgrade___Veterans_Affairs

## What areas of the site does it impact?
DUW v2

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/discharge-wizard/components/v2/resultsComponents/StepOne.jsx [hunk 1]
```diff
@@ -93,14 +93,13 @@ const StepOne = ({ formResponses }) => {
...
         {strongCaseTips()}
       </li>
-
       {[
         RESPONSES.PREV_APPLICATION_BCMR,
         RESPONSES.PREV_APPLICATION_BCNR,
       ].includes(formResponses[SHORT_NAME_MAP.PREV_APPLICATION_TYPE]) && (
         <li>
           Because you’re applying for reconsideration of a previous application,
-          you’ll need to enter the previous application number in Item 6b.{' '}
```
Comment (randimays) [inline src/applications/discharge-wizard/components/v2/resultsComponents/StepOne.jsx:103]: I think we might be missing part of the ACs? Could be wrong though. Line 125 might need to be changed to this:

Screenshot 2024-11-15 at 12 28 25 PM

Comment (chriskim2311) [reply src/applications/discharge-wizard/components/v2/resultsComponents/StepOne.jsx:103]: Ahh good catch thanks @randimays, my merge conflict removed my change. I'll add that back in

**Review by randimays — COMMENTED** [review]
I'm still seeing `item 6` in the review instance using the answers from your testing steps:

Screenshot 2024-11-18 at 7 18 23 AM

Does `item 6` need to be updated in these files as well?
- https://github.com/department-of-veterans-affairs/vets-website/blob/c5dcb04c53219638e242933ecbd0dff930d6799a/src/applications/discharge-wizard/components/v2/RequestDD214.jsx#L55
- https://github.com/department-of-veterans-affairs/vets-website/blob/c5dcb04c53219638e242933ecbd0dff930d6799a/src/applications/discharge-wizard/components/v2/resultsComponents/StepOne.jsx#L80

**Comment by chriskim2311** [comment]
@randimays Thanks for the sanity check. I have updated those files as well.

**Review by randimays — APPROVED** [review]
LGTM

**Review by harshilmoradia — APPROVED** [review]

**Pull request merged**